frontend/src/pages/NotFound/notFound.jsx

```diff
@@ -0,0 +1,34 @@
+import './notFound.css';
+import { useNavigate } from 'react-router-dom';
+
+function NotFound() {
+    const navigate = useNavigate();
+    const returnToHomePage = () => {
+        navigate('/');
+    };
+
+    return (
+        <div className={'not-found'}>
+            <section className={'not-found--container'}>
+                <h1 className={"not-found--title"}>404</h1>
+                <div className={"not-found--content"}>
+                    <h2 className={"not-found--subtitle"}>Щось пішло не так</h2>
+                    <p className={"not-found--description"}>
+                        Схоже, це неправильна адреса, ця сторінка видалена,
+                        перейменована або тимчасово недоступна.
+                    </p>
+                    <div>
+                        <button
+                            className={'button button__padding button__primary-color'}
```

Comment:
Great use of utility classes! If you find yourself repeating these combinations often, consider creating a reusable Button component to reduce duplication and improve consistency.